apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -724,59 +733,15 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
                 return (
                     <Segment basic >
                         <h5 className=" text-center"> { t(translateKey + "modals.scan.heading") }</h5>
-                        <Segment textAlign="center" basic className="qr-code">
-                            { qrCode
-                                ? <QRCode value={ qrCode } data-testid={ `${ testId }-view-modals-scan-qrcode` }/>
-                                : null
-                            }
-                        </Segment>
                         <div className="ml-3 mr-3">
-                            <Message className="display-flex" size="small" warning>
-                                <Icon name="warning sign" color="orange" corner />
-                                <Message.Content className="tiny">
-                                    { t(commonConfig.accountSecurityPage.mfa.totp.regenerateWarning) }
```

Comment:
Check the possibility of removing `regenerateWarning` config as well. Seems the usage has been removed this PR.